Libplanet/Store/IStateStore.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Immutable;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Blocks;
+
+namespace Libplanet.Store
+{
+    /// <summary>
+    /// An interface to store states.
+    /// </summary>
+    public interface IStateStore
+    {
+        /// <summary>
+        /// Sets states mapped as relation <see cref="Block{T}.Hash"/> → states.
```

Comment:
What does “mapped as relation …” mean?

Reply:
Sadly, I couldn't find well expression for it 😢. Actually I wanted to say like *Stores the key-value pair with blockHash as key, states as value.*